rdflib/term.py

```diff
@@ -1858,30 +1862,50 @@ def _castPythonToLiteral(
 _toPythonMapping.update(XSDToPython)
 
 
+def reset_bindings() -> None:
```

Comment:
Is there a need for this to be public? Would somewhat prefer to make it private if possible.

Reply:
sure, why not